admin.rst

```diff
@@ -120,10 +120,10 @@ The burst argument tells Nginx to start dropping requests if more than five queu
 
 Nginx rate limiting is general and indiscriminate. To rate limit each authenticated request individually you will need to add logic in a :ref:`Custom Validation <custom_validation>` function.
 
-.. _connection_poolers:
+.. _external_connection_poolers:
 
-Using Connection Poolers
-------------------------
+Using External Connection Poolers
+---------------------------------
 
 In order to increase performance, PostgREST uses prepared statements by default. However, this setting is incompatible with connection poolers such as PgBouncer working in transaction pooling mode. In this case, you need to set the :ref:`db-prepared-statements` config option to ``false``. On the other hand, session pooling is fully compatible with PostgREST, while statement pooling is not compatible at all.
```

Comment:
We need to clarify here that PostgREST already has its own pool, maybe linking to https://postgrest.org/en/v8.0/configuration.html#db-pool.

Also maybe mention "external":
```suggestion
In order to increase performance, PostgREST uses prepared statements by default. However, this setting is incompatible with external connection poolers such as PgBouncer working in transaction pooling mode. In this case, you need to set the :ref:`db-prepared-statements` config option to ``false``. On the other hand, session pooling is fully compatible with PostgREST, while statement pooling is not compatible at all.
```